scripts/rollup/bundles.js

```diff
@@ -216,15 +228,18 @@ const bundles = [
     bundleTypes: [FB_WWW_DEV, NODE_DEV, NODE_PROD, UMD_DEV, UMD_PROD],
     entry: 'react-dom/test-utils',
     global: 'ReactTestUtils',
+    minifyWithProdErrorCodes: false,
     externals: ['react', 'react-dom'],
   },
 
+  // TODO: We don't use this bundle anymore
```

Comment:
@acdlite What do you mean? The Test Selector API kind of stalled but to my knowledge, we still have plans to make use of it. (I'm thinking about potentially using it for DevTools e2e tests at the moment via #22760)